src/app/oauth.ts

```diff
@@ -0,0 +1,17 @@
+import ClientOAuth2 from 'client-oauth2';
+
+import {ENV, zendeskClientID, routes as r} from '../utils';
+import {getManifest} from '../../manifest';
+
+// TODO install the app via the zendesk API and store secret
```

Comment:
Is this TODO still applicable?

Reply:
yes.  I want to research the possibility of using the zendesk API to create oauth App instead of having a sysadmin do it manually.

Reply:
ticket created https://mattermost.atlassian.net/browse/MM-31907